share/completions/apt.fish

```diff
@@ -23,6 +23,20 @@ function __fish_apt_subcommand
     complete -f -c apt -n '__fish_apt_no_subcommand' -a $subcommand $argv
 end
 
+function __fish_apt_using_subcommand --description 'Test if given subcommand is used'
+    for i in (commandline -opc)
+        if contains -- $i $argv[1]
```

Comment:
Why `$argv[1]` rather than just `$argv`? If more than one value is passed in isn't that an error?